utils/TTAlpsABCDConfigHelper.py

```diff
@@ -3,7 +3,7 @@
 from Sample import Sample, SampleType
 from ttalps_samples_list import dasBackgrounds2018,dasBackgrounds2017,dasBackgrounds2016PreVFP,dasBackgrounds2016PostVFP
 from ttalps_samples_list import dasBackgrounds2022preEE,dasBackgrounds2022postEE,dasBackgrounds2023preBPix,dasBackgrounds2023postBPix
-from ttalps_samples_list import dasSignals2018,dasSignals2017,dasSignalsPrivate2018,dasSignals2016PreVFP,dasSignals2016PostVFP
+from ttalps_samples_list import dasSignals2018,dasSignals2017,dasSignals2016PreVFP,dasSignals2016PostVFP
 from ttalps_samples_list import dasSignals2022postEE,dasSignals2022preEE,dasSignals2023preBPix,dasSignals2023postBPix
```

Comment:
I started to always include these files in the same order, one line per year. You can see that, e.g., in histogrammer files config. Maybe we can stick to the same convention?